catalog/pipeline/release/0.1/tests/run.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: release-run-empty-params
+spec:
+  params:
+    - name: applicationSnapshot
+      value: ""
+    - name: policy
+      value: ""
+    - name: extraConfigGitUrl
+      value: ""
+    - name: extraConfigGitRevision
+      value: ""
+    - name: extraConfigPath
+      value: ""
+    - name: postCleanUp
+      value: ""
+  pipelineRef:
+    name: release
+    bundle: quay.io/release-service/pipeline-release:0.1@main
```

Comment:
how will this work in a CI run? will the bundles get overridden by newly built artifacts?

Reply:
Using this pipelineRef, yes.
It should pull the image tagged "0.1" and "main", but of course one could use a digest, or the GitHub repo's commit sha1.
For the sake of simplicity, this definition uses the version tag + branch-ref tag

Reply:
FYI - This pipeline was recently pushed.
Take a look here:
https://quay.io/repository/hacbs-release/pipeline-release?tab=tags

And, I see a mistake, so I'm going to push an updated commit. fyi `s/hacbs-release/release-service/`

Reply:
To finalize this conversation. 
We will be removing these extra tags.
We will be using only the catalog version (e.g., 0.2) of the resource, the image digest, or both.
(resolving)